ansible/roles/nginx/templates/connectbox_static-site.conf.j2

```diff
@@ -11,4 +11,9 @@ server {
 {{ nginx_chat_block }}
 {{ nginx_admin_block }}
 
+    # Admin interface
+    location /admin {
```

Comment:
The `auth_basic` and `auth_basic_user_file` parameters are missing here. I notice that the tests passed, and knowing one of them tests for a 404, I suspect somethign else is doing the auth. Is that correct? Can you please add a comment to that end if it's the case.

Reply:
`/admin` is no longer protected since it is hosted by the unified UI. The only protected url is the `/admin/api.py` path: https://github.com/ConnectBox/connectbox-pi/blob/master/ansible/group_vars/all#L60